package.json

```diff
@@ -1,13 +1,17 @@
 {
   "name": "typed-rest-client",
-  "version": "0.14.2",
+  "version": "0.15.0",
   "description": "Node Rest and Http Clients for use with TypeScript",
   "main": "./RestClient.js",
   "scripts": {
     "build": "node make.js build",
     "test": "node make.js test",
     "samples": "node make.js samples"
   },
+  "engines": {
+    "node": ">=8.9.0",
```

Comment:
node 8.9 / npm 5.1 is only a developer build time dependency.  At runtime this should and needs to work on 6.x LTS and even back to node 5.10.  I would remove this from the PR and make another issue to define runtime req (I want our CI to multi plex and validate on ideally >= node 4.x LTS but at least node >=6.x LTS)